.github/workflows/main.yml

```
@@ -13,72 +12,31 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-
-      - uses: actions/setup-go@v5
-        with:
-          go-version-file: 'go.mod'
-
-      - name: Build
-        run: go build ./...
+    uses: omec-project/.github/.github/workflows/build.yml@main
+    with:
+      branch_name: ${{ github.ref }}
 
   staticcheck:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-
-      - uses: WillAbides/setup-go-faster@v1.14.0
-        with:
-          go-version-file: 'go.mod'
-
-      - uses: dominikh/staticcheck-action@v1.3.1
-        with:
-          version: latest
-          install-go: false
+    uses: omec-project/.github/.github/workflows/static-analysis.yml@main
+    with:
+      branch_name: ${{ github.ref }}
 
   lint:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-
-      - uses: actions/setup-go@v5
-        with:
-          go-version-file: 'go.mod'
-
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@v7.0.0
-        with:
-          version: latest
-          args: -v --config ./.golangci.yml
+    uses: omec-project/.github/.github/workflows/lint.yml@main
+    with:
+      branch_name: ${{ github.ref }}
 
   license-check:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-
-      - name: reuse lint
-        uses: fsfe/reuse-action@v5
+    uses: omec-project/.github/.github/workflows/license-check.yml@main
+    with:
+      branch_name: ${{ github.ref }}
 
   fossa-check:
```

This was renamed in all other repos
```suggestion
  fossa-scan:
```